build-tools/xaprepare/xaprepare/Steps/Step_Get_Windows_Binutils.cs

```diff
@@ -79,26 +79,32 @@ public Step_Get_Windows_Binutils ()
 		protected override async Task<bool> Execute (Context context)
 		{
 			string ndkVersion = BuildAndroidPlatforms.AndroidNdkVersion;
+			string baseArchivePath = $"android-ndk-r{ndkVersion}/toolchains/llvm/prebuilt/windows-x86_64/bin";
 
-			var neededFiles = new HashSet<string> (StringComparer.OrdinalIgnoreCase) {
```

Comment:
I don't understand why the `HashSet<string>` was replaced with `Dictionary<string, string>`.  Nor do I understand what the "value" is supposed to be…

Reply:
ah, "value" is "destination path".

Reply:
Key is the original file name, Value is an optional target file name. Used to move `ld.gold` to `ld`